src/main/php/org/bovigo/vfs/DotDirectory.php

```diff
@@ -1,35 +1,36 @@
 <?php
+
 declare(strict_types=1);
+
 /**
  * This file is part of vfsStream.
  *
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
- *
- * @package  org\bovigo\vfs
  */
+
 namespace org\bovigo\vfs;
+
+use ArrayIterator;
+use Iterator;
+
 /**
  * Directory container.
  */
 class DotDirectory extends vfsStreamDirectory
 {
     /**
      * returns iterator for the children
-     *
-     * @return  \Iterator
      */
-    public function getIterator(): \Iterator
+    public function getIterator() : Iterator
```

Comment:
For some reason we decided to use doctrine coding standard. This forces that you are add a use statement all classes, functions and constants. It has been proved solution that resolving functions and constants is slow in php. So I prefer to have them as use statements. because it is hard to force the `\` in front of functions. 

I think it is easier to add all classes in a use statement that distinguish between global and non global classes. I don't know if it does any harm to have them in a use statement.

Reply:
For the use statements, I don't think it matters if they're there or not. It's all preference. If there's a sniff that fixes it to be one way or the other, it probably doesn't matter.

For the spacing around the `:` in the return type, I definitely think we should try to follow the PSRs and write it as `getFoo(): Foo`. PSRs aren't perfect, but they are the closest thing to having a community standard. If the Doctrine standard is using the [slevomat rule](https://github.com/slevomat/coding-standard#slevomatcodingstandardtypehintsreturntypehintspacing-) for this, we should be able to set `spacesCountBeforeColon` so that it matches the PSR.